portfolio-edit/Panel.php

```diff
@@ -10,67 +10,43 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+<link rel="stylesheet" href="panel.css">
 <body>
 
 <h1> Admin page </h1>
+
+<body>
```

Comment:
dont need a body tag here. not allowed 2 body tags

Reply:
done

Reply:
cool